content/doc/book/system-administration/backing-up.adoc

```diff
@@ -97,7 +98,29 @@ Backup retention is based on file names that include both the job name and the b
 
 One of the most important things to remember when it comes to your Jenkins backup is to test your backup restore configuration on a Jenkins test server.
 
-== Using Jenkins backup plugins
+== How to back up Jenkins data
+
+=== File-based backup tools
+
+The tar command is widely used to create compressed archive files that can be moved easily from one disk to another disk, or machine to machine. Tar stands for tape archive, and is commonly referred to as tarball or tar, gzip and bzip in Linux. 
+
+===== tar syntax
+- c – Creates a new .tar archive file.
+- z – use a gzip compression on the .tar archive
+- f – File name type of the archive file.
+
+From the command line, change directories into the `$JENKINS_HOME` folder. 
+----
+$ cd $JENKINS_HOME
+===== Create tar.gz Archive File
+----
+----
+$ tar czf backup_archive.tgz backupdirectory/
+----
+
+To restore from the backup_archive, in the Jenkins GUI, navigate to the "Manage Jenkins" page and choose the "Reload Configuration from Disk" option.
```

Comment:
That's not what this command does.

In general, tar backups should not be restored while Jenkins is running.

Reply:
My misunderstanding, I was looking at the archive section here: https://wiki.jenkins.io/display/JENKINS/Administering+Jenkins and thought it might apply to archiving the Jenkins_Home directory. 

So for tar backups, you recommend to stop/restart a Jenkins instance?

Reply:
To restore, definitely. They can be taken while it's running, if we tolerate a minor amount of inconsistency.